app/models/mainstream_browse_page.rb

```diff
@@ -31,7 +31,12 @@ def base_path
     "/browse#{super}"
   end
 
+  def tag_type
```

Comment:
Would be good to rename this method to something like `legacy_tag_type` to make it clear that it's only applicable to the old-world stuff.